pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/kubernetes/KubernetesRuntimeFactory.java

```diff
@@ -104,7 +109,7 @@
     private SecretsProviderConfigurator secretsProviderConfigurator;
     @ToString.Exclude
     @EqualsAndHashCode.Exclude
-    private Optional<KubernetesFunctionAuthProvider> authProvider;
+    private Optional<String> authProviderClassName;
```

Comment:
Why do we need to change this?

Reply:
See comment here: https://github.com/apache/pulsar/pull/5400/files#r344416501